source/platforms/_tests/_bitbucket_cloud.test.ts

```diff
@@ -75,4 +75,30 @@ describe("getPlatformReviewDSLRepresentation", () => {
     const dsl = await bbs.getPlatformReviewDSLRepresentation()
     expect(dsl.metadata).toMatchSnapshot()
   })
+
+  it("should update status", async () => {
+    api.postBuildStatus = jest.fn().mockReturnValue({ ok: true })
+    await bbs.updateStatus(true, "message", undefined, "danger", undefined)
+
+    expect(api.postBuildStatus).toHaveBeenCalledWith("007bf2423436", {
```

Comment:
Given this test was missing in both the platform I added it also if doesn't directly test the change in this PR